test/test_results.py

```diff
@@ -138,7 +138,7 @@ def test_get_result_hash_from_prefix(tmpdir):
     shutil.copyfile(machine_json, join(str(machine_dir), 'machine.json'))
 
     for f in ['e5b6cdbc', 'e5bfoo12']:
-        open(join(str(machine_dir), '{0}-py2.7-Cython-numpy1.8.json'.format(f)), 'a').close()
+        open(join(str(machine_dir), f'{f}-py2.7-Cython-numpy1.8.json'), 'a').close()
```

Comment:
Nit: it is a bit weird to be using these old versions of python (line 39?) and numpy (dunno where that comes from). If not too much trouble could you update them? If not no big deal.